src/golang/objectprotection/rubrik_snappable_protection_status.go

```diff
@@ -0,0 +1,92 @@
+package objectprotection
+
+import (
+	"log"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/rubrikinc/rubrik-sdk-for-go/rubrikcdm"
+)
+
+var (
+	// Rubrik Snappable SlaDomain Information
+	snappableEffectiveSlaDomain = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "rubrik_snappable_effective_sla",
+			Help: "Return the slaDomain information for snappables",
+		},
+		[]string{
+			"clusterName",
+			"objectName",
+			"objectType",
+			"objectID",
+			"location",
+			"slaDomain",
+		},
+	)
+)
+
+func init() {
+	// VMware vSphere VM effective sla domain
+	prometheus.MustRegister(snappableEffectiveSlaDomain)
+}
+
+// GetSnappableEffectiveSlaDomain ...
+func GetSnappableEffectiveSlaDomain(rubrik *rubrikcdm.Credentials, clusterName string) {
+	reportData, err := rubrik.Get("internal", "/report?report_template=ObjectProtectionSummary&report_type=Canned", 60) // get our object protection summary report
+	if err != nil {
+		log.Printf("Error from objectprotection.GetVSphereVmEffectiveSlaDomain: ", err)
+		return
+	}
+	reports := reportData.(map[string]interface{})["data"].([]interface{})
+	reportID := reports[0].(map[string]interface{})["id"]
+	body := map[string]interface{}{
+		"limit": 100,
+	}
+	for {
+		hasMore := true
+		tableData, err := rubrik.Post("internal", "/report/"+reportID.(string)+"/table", body, 60) // get our first page of data for the report
+		if err != nil {
+			log.Printf("Error from objectprotection.GetVSphereVmEffectiveSlaDomain: ", err)
```

Comment:
Looks like a copy 'n paste related accident.

Reply:
I.e. `objectprotection.GetVSphereVmEffectiveSlaDomain` in the log message.

Reply:
Corrected in commit https://github.com/rubrikinc/rubrik-client-for-prometheus/pull/21/commits/18df917b972817cc08e86ec4d633ba22acd7d164